e2e/verifyocsp/verify-ocsp.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2019-2022, Sylabs Inc. All rights reserved.
```

Comment:
This should be the appropriate copyright statement for you, and dated this year.